lib/active_admin/inputs/datepicker_input.rb

```diff
@@ -1,6 +1,8 @@
 module ActiveAdmin
   module Inputs
     class DatepickerInput < ::Formtastic::Inputs::StringInput
+      include ::Formtastic::Inputs::Base
```

Comment:
I don't think this inclusion is needed?

Reply:
You are right, `Formtastic::Inputs::StringInput` already includes `Base`